app/javascript/packs/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Body from './Body'
+import MyHeader from './MyHeader'
+
+class App extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            cuisines: []
+
+        };
+    }
+
+    componentWillMount() {
+        fetch('http://localhost:3000/cuisines.json')
```

Comment:
1. Try extracting your `fetch`es to a utility module since you use it repeatedly (`.json()` etc)
2. You shouldn't hard-code the server hostname, that won't work in production
3. Don't forget to handle your rejected promises (`.catch`)

Reply:
Thanks!!!!!! updated (: